content/release-10-16-0/platform-10-16-0-bundle/10-16-0-0.md

```diff
@@ -185,12 +163,40 @@ MTM-48529</td>
 <td>MTM-48344</td>
 </tr>
 
+<tr>
+<td>REST API</td>
+<td>The performance of the Alarm API has been improved for requests in which the resolved query parameter is used.</td>
+<td>MTM-49450</td>
+</tr>
+
 <tr>
 <td>Security</td>
 <td>An attribute was added to all external application links, that prevents tab nabbing, a security vulnerability that could be used to gain the user's browser session.</td>
 <td>MTM-48156</td>
 </tr>
 
+<td>
+Single sign-on</td>
+<td>Trial tenants created in Software AG Cloud are by default not allowed to access the single sign-on configuration in Cumulocity IoT to prevent its potential misconfiguration by users. A REST endpoint is used by the Management tenant to restrict or allow the access to the configuration for specific tenants. Refer to <a href="https://cumulocity.com/guides/10.16.0/users-guide/administration/#configuration-access">Administration > Single sign-on > Configuration access</a> in the <i>User guide</i> for more details. </td>

```

Comment:
is "Software AG Cloud" a product name? (asking because of capital "Cloud")

Reply:
Yes, see https://www.softwareag.cloud/site/dev-center/cumulocity-iot.html#/